.gitignore

```diff
@@ -1 +1,2 @@
+*.iml
```

Comment:
Please, could you exclude this line from the project gitignore. Good rule is to keep your IDE ignore files in the global gitignore, see `core.excludesfile` for `git config --help` for more details